.github/workflows/validation.yml

```diff
@@ -23,25 +23,10 @@ jobs:
         make install-tools
     - name: run build
       run: |
-        make build
-    - name: run unit-test
+        make manager
+    - name: run unit tests and E2E tests (fake cluster)
       run: |
         make test
-    - name: run validate
```

Comment:
do we have an equivalent command? is it included in `make manager` ?

Reply:
Yes `manager` depends on `lint`